File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int sum = 0;
        for (int i = 0; i < s.length(); i++) {
            int rev_idx = ('z' - s[i]) + 1;
            cout<<i+1<<" "<<rev_idx<<endl;
            sum += (i+1) * rev_idx;
        }
        return sum;
    }
};

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.java
class Solution {
    public int reverseDegree(String s) {
      int degree = 0;
        for (int i = 0; i < s.length(); i++) {
            int val = 26 - ((int)s.charAt(i) - 'a');
            degree += (i+1) * val;
        }
        return degree;
    }
}